index.js

```diff
@@ -101,9 +102,8 @@ function commit (argv, newVersion, cb) {
     args.unshift('package.json')
   }
   checkpoint(argv, msg, args)
-
   handledExec(argv, 'git add package.json ' + argv.infile, cb, function () {
-    handledExec(argv, 'git commit ' + verify + (argv.sign ? '-S ' : '') + 'package.json ' + argv.infile + ' -m "' + formatCommitMessage(argv.message, newVersion) + '"', cb, function () {
+    handledExec(argv, 'git commit ' + verify + (argv.sign ? '-S ' : '') + (argv.commitAll ? '' : ('package.json ' + argv.infile)) + ' -m "' + formatCommitMessage(argv.message, newVersion) + '"', cb, function () {
```

Comment:
This line is a bit messy. I'd like to use a variable. But doesn't have to be in this PR :)